framework/include/mediaplayer/mediaplayer.h

```diff
@@ -0,0 +1,213 @@
+/****************************************************************************
```

Comment:
Why is this media player instead of audio player?

Reply:
I guess there is not much difference between media and audioplayer. "Mediaplayer" name is used by android and even by windows to play media files.